python/ray/experimental/serve/server.py

```diff
@@ -128,13 +128,29 @@ def __init__(self):
         endpoint_name = self.route_table_cache[current_path]
         http_body_bytes = await self.receive_http_body(scope, receive, send)
 
+        # get slo_ms before enqueuing the query 
+        query_string = scope["query_string"].decode("ascii")
+        query_kwargs = parse_qs(query_string)
+        request_slo_ms =  query_kwargs.pop('slo_ms',None)
+        if request_slo_ms is not None:
+            request_slo_ms = request_slo_ms[0]
+            try:
+                request_slo_ms = float(request_slo_ms)
+                if request_slo_ms < 0:
+                    raise Exception
+            except:
+                await JSONResponse({
+                    "error": "slo_ms specified wrongly!"
+                })(scope, receive, send)
+                return
+
```

Comment:
We should explicitly catch the error instead of using bare `except`. And give meaningful error message. 
```python
if request_slo_ms is not None:
    if len(request_slo_ms) != 1:
        raise ValueError("Multiple SLO specified, please specific only one.")
    request_slo_ms = float(request_slo_ms[0])
    if request_slo_ms <= 0:
        raise ValueError("Request SLO must be positive, it is {}".format(request_slo_ms))
    except ValueError as e:
        await JSONResponse({"error": str(e)})(scope, receive, send)
        return
```